service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/storage/DeleteAzureStorageStep.java

```diff
@@ -0,0 +1,65 @@
+package bio.terra.workspace.service.resource.controlled.cloud.azure.storage;
+
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.StepStatus;
+import bio.terra.workspace.app.configuration.external.AzureConfiguration;
+import bio.terra.workspace.service.crl.CrlService;
+import bio.terra.workspace.service.workspace.flight.WorkspaceFlightMapKeys.ControlledResourceKeys;
+import bio.terra.workspace.service.workspace.model.AzureCloudContext;
+import com.azure.resourcemanager.storage.StorageManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A step for deleting a controlled Azure Storage Account resource.
+ */
+public class DeleteAzureStorageStep implements Step {
+  private static final Logger logger = LoggerFactory.getLogger(CreateAzureStorageStep.class);
```

Comment:
```suggestion
  private static final Logger logger = LoggerFactory.getLogger(DeleteAzureStorageStep.class);
```

Reply:
I followed the pattern of other Delete steps, which are using the Create class for logger factory. For example:

```
public class DeleteAzureIpStep implements Step {
  private static final Logger logger = LoggerFactory.getLogger(CreateAzureIpStep.class);
```

@ddietterich was this intentional?